CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
The new 'Salary' property is added after the 'ForeignKey' attribute which was previously meant for 'CompanyId'. This disrupts the expected placement and may introduce EF mapping issues. Declare 'Salary' independently of the ForeignKey section.

```suggestion
public decimal Salary { get; set; }
```